components/x-audio/src/components/styles.scss

```diff
@@ -218,3 +223,20 @@ $info-height--expanded: 72px;
 	align-self: center;
 	margin-left: 8px;
 }
+
+.audio-player__status {
+  grid-area: status;
```

Comment:
@benbarnett this added part using spaces like my PR earlier.
I think the original scss file was using spaces instead of tabs.

Reply:
Well spotted, thanks, will update